extensions/emmet/src/test/tagActions.test.ts

```diff
@@ -117,7 +117,7 @@ suite('Tests for Emmet actions on html tags', () => {
 	</div>
 	`;
 		const oldValueForSyntaxProfiles = workspace.getConfiguration('emmet').inspect('syntaxProfiles');
-		return workspace.getConfiguration('emmet').update('syntaxProfiles', {jsx: {selfClosingStyle: 'xhtml'}}, ConfigurationTarget.Global).then(() =>{
+		return workspace.getConfiguration('emmet').update('syntaxProfiles', { jsx: { selfClosingStyle: 'xhtml' } }, ConfigurationTarget.Global).then(() => {
```

Comment:
The formatter kept on requiring I add these. Is there a good way to disable this?